support-frontend/package.json

```diff
@@ -10,6 +10,7 @@
     "build-dev": "webpack --config webpack.dev.js",
     "build-ssr": "webpack --config webpack.ssr.js && node scripts/ssrScript.js",
     "build-prod": "npm-run-all clean validate test storybook-static 'webpack --config webpack.prod.js --env.prod'",
+    "build-github-action": "yarn clean && CI_ENV=github webpack --config webpack.prod.js --env.prod",
```

Comment:
should we add a CI_ENV=teamcity for completeness above?

Reply:
Happy to add this, could be useful to make the distinction at some point.